people/views.py

```diff
@@ -172,6 +176,17 @@ class FarmersCsvAPIView(APIView):
     authentication_classes = [TokenAuthentication]
     permissions_classes =[IsAuthenticated]
 
+    def limitQueryset(self, queryset, start_limit, end_limit):
+        # limits the total response count        
+        if start_limit and end_limit: # case1: both are present
+            queryset = queryset[int(start_limit)-1:int(end_limit)]
+        elif start_limit: # case2: only start_limit is present
+            queryset = queryset[int(start_limit)-1:]
+        elif end_limit: # case3: only end_limit is present
+            queryset = queryset[:int(end_limit)]
```

Comment:
create a class for this and use it everywhere instead of repeating the code

Reply:
Code changed